### DISCUSSION TITLE
Compatibility of all wired setup

### DISCUSSION THREAD

**Discussion by ProtoxiDe22** [petrleocompel/hikaxpro_hacs#119] [Q&A] [answered]
Hello, I'm currently investigating my best options for a wired alarm system, the hikvision axpro hybrid looks like a really food option for me at this time, my setup would be comprised of all wired sensor, both PIR and door contacts, probably generic, non hikvision ones, which as far as i can tell are supported on the alarm lanel with eol resistors and all the bells and whistles

My question is how is a setup like this supported by this integration. Would i be able to see the alarm state of the zones in HA? Can i completely remove the need of hikvision keypads? (I have HA dashboard tablets that i would use to arm the alarm system)

**Comment by petrleocompel** [comment]
Probably it would work.

Wired sensors are displayed as this.
![image](https://github.com/user-attachments/assets/f9795928-43f9-454c-9b90-e7dfafa4eae9)

And you can see even separated zones. (grouped sensors in their UI = zones) In setting you can find an option for this and enable separate zones control in HA.

**Reply by petrleocompel** [reply]
I cannot tell what entities works how. That is what we get from "Hikvision". If they work for wired sensors the same way.

**Reply by ProtoxiDe22** [reply]
Even if I don't get the zone entities,, i will still be able to arm/disarm and get alarm events right? These should be handled by the alarm panel entities if understand it correctly

**Reply by petrleocompel** [reply, marked as answer]
Yes, just count with "delay" defined by pull interval. Because integration is continually pulling data from the device. So if it gets triggered and your "pull interval" in HA integration settings is 30 sec (default) it would take max 30 sec to display it in HA. (you can lower it - i know about people who has 1 second in config) but technically it could cause problems. There is still #2 (basically event trigger from device to HA) but that is still unsolved.

**Reply by ProtoxiDe22** [reply]
Thank you
